src/schema/v2/artwork/collectorSignals.ts

```diff
@@ -107,6 +107,25 @@ export const CollectorSignals: GraphQLFieldConfig<any, ResolverContext> = {
         description: "Bid count on lots open for bidding",
         deprecationReason: "Use nested field in `auction` instead",
       },
+      curatorsPick: {
+        type: GraphQLBoolean,
+        description:
+          "Artwork is part of either the Curators' Pick Emerging or Blue Chip collections",
+        resolve: async (artwork, {}, ctx) => {
+          const CURATED_COLLECTION_SLUGS = [
+            "curators-picks-blue-chip-artists",
+            "curators-picks-emerging-artists",
+          ]
+
+          const checks = await Promise.all(
+            CURATED_COLLECTION_SLUGS.map(async (slug) => {
+              const collection = await ctx.marketingCollectionLoader(slug)
+              return collection.artwork_ids.includes(artwork._id)
+            })
+          )
+          return checks.includes(true)
```

Comment:
I'm worried that this might slow down the query, due to sending 2 requests per artwork in the artworks array

Should we get the list and compare them locally instead?

Reply:
We've generally agreed to do this on MP in part because the [dataloader](https://github.com/graphql/dataloader) package will avoid duplicative requests when resolving each query. So this should not be an issue if the request for each artwork is the same.

Reply:
These are parallelized via `Promise.all` and so it's not clear if combining them into one, potentially _slower_ request is actually better. I personally like to stick to simpler requests like this - then, if performance seems to be an issue, there are tools at our disposal:
  - longer lived cache
  - denormalization